lib/babel-config.js

```diff
@@ -14,9 +12,9 @@ function validate(conf) {
 	const isValidShortcut = conf === 'default' || conf === 'inherit';
 
 	if (!conf || (typeof conf === 'string' && !isValidShortcut)) {
-		let message = colors.error(figures.cross);
+		let message = figures.cross;
 		message += ' Unexpected Babel configuration for AVA. ';
-		message += 'See ' + chalk.underline('https://github.com/avajs/ava#es2015-support') + ' for allowed values.';
+		message += 'See https://github.com/avajs/ava#es2015-support for allowed values.';
```

Comment:
@sindresorhus you suggested to remove the color comparison from the test, but this removes it from the actual AVA code.

Why do the changes in this PR require colors to be removed elsewhere? Should we then also remove colors from https://github.com/avajs/ava/blob/master/lib/cli.js#L100:L108?